compose.c

```diff
@@ -862,6 +902,42 @@ static int draw_envelope_addr(int field, struct AddressList *al,
   return lines_used;
 }
 
+static int draw_envelope_user_hdrs(const struct ComposeRedrawData *rd, int row)
```

Comment:
Please can you add a Doxygen comment block for this function.

Reply:
Not sure how I missed that one, added with 4736e68